.pre-commit-config.yaml

```diff
@@ -22,11 +22,6 @@ repos:
     - --autofix
   - id: trailing-whitespace
 
-- repo: https://github.com/dnephin/pre-commit-golang
-  rev: v0.3.5
-  hooks:
-  - id: go-fmt
```

Comment:
Why is this removed?
Is it now included with the version upgrade?

Reply:
Hey @thetechnick, gofumpt is already a part of golangci-lint, it uses a superset of what gofmt enforces.